pkg/discovery/probe/pod_probe_util.go

```diff
@@ -56,6 +56,27 @@ func GetResourceRequest(pod *api.Pod) (cpuRequest float64, memRequest float64, e
 	return
 }
 
+// Get the UUID that will be used in Turbonomic server. Here we build the UUID based on pod UID, namespace and name.
+// The out UUID is in format "UID:namespace:name"
+func GetTurboPodUUID(pod *api.Pod) string {
+	uid := string(pod.UID)
```

Comment:
any document about pod.UID? Is it unique in one kubernete cluster, or even unique in multiple kubernete clusters?